cf-agent/files_properties.c

```diff
@@ -48,32 +48,17 @@ static const char *const SKIPFILES[] =
 static bool ConsiderFile(const char *nodename, const char *path, struct stat *stat)
 {
     int i;
-    
-    if (stat != NULL)
-    {
-        if (S_ISLNK(stat->st_mode))
-        {
-            Log(LOG_LEVEL_INFO, "'%s' is a symbolic link", nodename);
-        }
-        else if (S_ISDIR(stat->st_mode))
-        {
-            Log(LOG_LEVEL_INFO, "'%s' is a directory", nodename);
-        }
-        
-        Log(LOG_LEVEL_VERBOSE, "'%s' has size %ld and full mode %o", nodename, (unsigned long) (stat->st_size),
-              (unsigned int) (stat->st_mode));
-    }
 
     if (strlen(nodename) < 1)
     {
-        Log(LOG_LEVEL_ERR, "Empty (null) filename detected in '%s'", path);
+        Log(LOG_LEVEL_ERR, "Empty (null) filename detected in '%s', skipping", path);
         return false;
     }
 
     if (stat != NULL && (S_ISREG(stat->st_mode) || S_ISLNK(stat->st_mode)) &&
         IsItemIn(SUSPICIOUSLIST, nodename))
     {
-        Log(LOG_LEVEL_ERR, "Suspicious file '%s' found in '%s'", nodename, path);
+        Log(LOG_LEVEL_ERR, "Filename '%s/%s' is classified as suspiscious, skipping", path, nodename);
```

Comment:
https://github.com/cfengine/core/blob/master/libpromises/mod_common.c#L311 says:

> "suspiciousnames", "", "List of names to warn about if found during any file search"

I understand it as a WARN message and no skipping, which is not the current behavior. We can:
* Make it a WARN log message and keep the file, like it is documented
* Make it a WARN message and skip the file
* Make it a INFO/VERBOSE message and skip the file

It think we should use the WARN level, because it is a list of "suspicious" files and not simply an exclusion list, and the user probably wants to know when a file considered suspicious is found.
I don't know if we should should skip or not (i.e. if we should break the behavior or the specs), what do you think?